css/styleEx.css

```diff
@@ -0,0 +1,327 @@
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+    width: 100%;
+    margin: auto;
+}
+
+/* Header Styles */
+
+header {
+    width: 100%;
+    display: flex;
+    
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+    display: flex;
+}
+
+header nav a {
+    text-decoration: none;
+    color: #b0b0b0;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 16px;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+   
+   
+}
+
+.main-content .top-content {
+    display: flex;
+
+    width: 80%;
+    justify-content: center;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+
+    width: 80%;
+}
+
+.main-content .bottom-content {
+    display: flex;
+
+    width: 80%;
+    margin: auto;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+@media only screen and (max-width: 800px) {
+    header {
+        width: 100%;
+        flex-wrap: nowrap;
+        display: flex; 
+        flex-direction: column-reverse;
+        /* justify-content: center; */
+        /* width: 100%; */
+        align-items: center;
+    }
+    /* header nav a {
+        width: 100%;
+        /* display: flex;
+        justify-content: center; 
+        align-content: center;*/
+    
+    header .logo {
+        width: 25%;        
+        display: flex;         
+        justify-content: center; 
+        align-items: center;
+                    
+    }
+    .cta .cta-text {
+        width: 100%;
+        /* align-content: center; */
+        display: flex;         
+        justify-content: center; 
+        align-items: center;
+
```

Comment:
I see what you were trying to do here. Making the text more centered. Some `margin-left` would have done it.